### DISCUSSION TITLE
Would it be possible to use it in Angular

### DISCUSSION THREAD

**Discussion by k3nsei** [chakra-ui/panda#753] [Q&A] [answered]
As this CSS in JS solution is based on PostCSS. Would be possible to integrate it somehow with Angular? As generation of styles is not based on JSX and result of functions are strings with classes.

**Comment by segunadebayo** [comment, marked as answer]
Hi @k3nsei,

Yes, it should work in Angular if PostCSS is supported there. Please follow the steps in the docs and let us know if you have it working.

Happy to update the docs to include it.

**Comment by k3nsei** [comment]
@segunadebayo I managed to make it work with watch mode as Angular CLI is not exposing ability to override postcss config. It would be possible to do it with NX as alternative to Angular CLI.

Here is example app https://github.com/k3nsei/ng-panda-css

To build it just run `npm install` and then `npm run build`.

To start dev server `npm install` and then `npm run start`.

**Comment by k3nsei** [comment]
@segunadebayo let me know if you need any explanation or help with further work that needs to be done.

**Comment by k3nsei** [comment]
I also created feature request to add support for it in Angular CLI https://github.com/angular/angular-cli/issues/25411 But I have doubts if they would make it reality.

**Comment by k3nsei** [comment]
@segunadebayo Angular CLI team picked this issue as it could be done in future. But now is under voting, it requires 20 votes to move forward. So if you can share it and ask people to vote on it https://github.com/angular/angular-cli/issues/25411

**Comment by k3nsei** [comment]
As Angular v17.2.0 has support for custom postcss config it work with PandaCSS now without any problems.

https://github.com/angular/angular-cli/commit/7c522aa8742cd936bb0dfd30773d88f3ef92d488

**Reply by segunadebayo** [reply]
That’s amazing to hear 🥳
